File: zendrive-ui/src/api/models/stats/FileStats.ts
export class FileStats {
	fileCount: number;
	directoryCount: number;
	totalSize: number;

	constructor(fileCount: number, directoryCount: number, totalSize: number) {
		this.fileCount = fileCount;
		this.directoryCount = directoryCount;
		this.totalSize = totalSize;
	}
}

File: zendrive-api/src/main/java/com/zendrive/api/core/model/metafile/FileStats.java
package com.zendrive.api.core.model.metafile;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class FileStats {
	public int fileCount = 0;
	public int directoryCount = 0;
	public long totalSize = 0;
}
